File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.py
changes = {
    1: "w",
    -1: "s",
    10: "d",
    -10: "a"
}

def solution(numLog):
    answer = ''
    for i in range(len(numLog) - 1):
        answer += changes[numLog[i+1] - numLog[i]]
    return answer

File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.js
const CHANGES = {
    1: "w",
    "-1": "s",
    10: "d",
    "-10": "a"
}

function solution(numLog) {
    let answer = '';
    
    for (let i = 0; i < numLog.length - 1; i++) {
        const diff = numLog[i+1] - numLog[i];
        answer += CHANGES[diff];
    }
    
    return answer;
}